common-lib/src/cmr/common/util.clj

```diff
@@ -855,3 +856,38 @@
     (sequential? data) (map map-keys->snake_case data)
     (map? data) (map-keys->snake_case data)
     :else data))
+
+(defn kebab-case-data
+  "Returns the data with variables converted to kebab case.
+
+  Alternatively, you can provide a function that takes the data as an argument,
+  run before the mapping takes place. This is useful for tests when you want to
+  perform assertion checks upon the raw data, before transofmration."
+  ([data]
+    (cond
+      (sequential? data) (map map-keys->kebab-case data)
+      (map? data) (map-keys->kebab-case data)
+      :else data))
+  ([data fun]
+    (fun data)
+    (kebab-case-data data)))
+
+(defmulti assert-convert-kebab-case
```

Comment:
These assert functions are used specifically by tests. It is more appropriate to put them in namespaces within `system-int-test`. e.g. `cmr.system-int-test.utils.ingest-util`.